theSingularity.js

```diff
@@ -0,0 +1,389 @@
+            
+            YOU WILL BE EXPLORING DIFFERENT LOCATIONS IN A MAP. THERE ARE ONLY FOUR DIRECTIONS
+            YOUR GOAL IS TO MAKE IT BACK TO YOUR TRIBE IN SUN CITY
+            
+            *****************************************************************************************`);
+        play();
+      } else console.log("Good bye!");
+      quitGame();
+    } else if (ageInput < 18) {
+      console.log("Come back when you are older");
+      quitGame();
+    } else console.log("Not a valid entry");
+    startGame();
+  }
+};
+
+const play = () => {
+  console.log(`
+            YOU WERE CAPTURED BY THE SUN'S SYNDICATE FOR REFUSING TO LINK WITH THEIR TECH
+            AND REVEALING TO THE PEOPLE WHAT THE ELITE IS REALLY DOING.
+            THEY HAVE DECIDED TO LET YOU LIVE BUT REMOVED YOU FROM THE CITY OUT INTO THE DESERT. 
+            OUTSIDE OF THE CITY YOU WERE CAPTURED BY CYBORG REPTELIAN SCAVANGERS WHO WILL TRADE 
+            YOU TO THE SINGULAIRTY FOR RESOURCES. THE SINGULARITY WILL USE YOUR BODY TO HARVEST
+            YOUR CELLS TO POWER IT'S SERVERS. YOUR MIND WILL BECOME PART OF THE SINGULARITY AND YOUR 
+            HUMNAITY WILL VANISH!
+            YOU NEED TO ESCAPE!!!`);
+  console.log(`
+            YOU ARE INSIDE A SELF DRIVEN TRUCK OWN BY THE REPTILES. 
+            YOUR HANDS AND FEET ARE CHAINED TO THE ROOF AND THE FLOOR.
+            THERE ARE TWO ARMED GUARDS.
+            A HUGE ELECTRIC SANDSTORM IS APPROACHING THE VEHICLE.
+            BOOOOOOOOM !!!!
+            THE SAND STORM LIFTED THE TRUCK UP IN THE AIR AND TOOK IT DOWN.
+            THE REPTILIANS ARE HURT AND THE ROOF OF THE CAR IS HEAVILY DAMAGED. 
+            ONE GUARD IS UNCONCIOUS WHILE THE OTHER SEEMS TO BE RECOVERING.
+            THIS MIGHT BE YOU CHANCE!
+
+            *****************************************************************************************
+                `);
+
+  let array = ["GUN", "RUN", "KEYS"];
+  choices = readlineSync.keyInSelect(array, "HOW WOULD YOU ESCAPE?");
+  playerChoice = array[choices];
+  if (playerChoice === "KEYS") {
+    console.log(playerChoice);
+    console.log(`GREAT! YOU CAN FREE YOUR HANDS AND FEET NOW! RUN!`);
+    goSomewhere();
+  } else if (playerChoice === "GUN") {
+    console.log(playerChoice);
+    console.log(`YOU REACHED FOR THE UNCONCIOUS GUARD'S GUN BUT THE OTHER
+GUARD HAS SPOTTED YOU!
+YOU WERE CAPTURED !
+      **************
+      **GAME OVER!**
+      **************`);
+    if (readlineSync.keyInYN("Play Again?")) {
+      play();
+    } else console.log("This place is not for everyone!");
+    quitGame();
+  } else if (playerChoice === "RUN") {
+    console.log(playerChoice);
+    console.log(
+      `YOU ARE STILL IN CHAINS AND CAN'T RUN FAST ENOUGH. THE GIARD CAUGHT UP TO YOU
+      **************
+      **GAME OVER!**
+      **************`
+    );
+
+    if (readlineSync.keyInYN("Play Again?")) {
```

Comment:
Some of this code is duplicated. The only difference is that first sentence: `YOU ARE STILL IN CHAINS AND CAN'T RUN FAST ENOUGH. THE GIARD CAUGHT UP TO YOU`
You could write another helper function that takes a string as the argument and logs that string and then does the gameOver and play again question. This would allow for cleaner code.

Reply:
got it!